src/librados/librados.cc

```diff
@@ -2423,13 +2426,19 @@ int librados::Rados::pool_create(const char *name)
 int librados::Rados::pool_create(const char *name, uint64_t auid)
 {
   string str(name);
-  return client->pool_create(str, auid);
```

Comment:
yeah good idea.  i went with EINVAL though since it's more like 'invalid input'.. it seems weird to have a EOPNOTSUPP return value that depends on the arguments passed.